test/FillUtils.ts

```diff
@@ -62,7 +63,9 @@ describe("FillUtils", function () {
         updatedOutputAmount: deposit.outputAmount,
         updatedRecipient: deposit.recipient,
         fillType: FillType.FastFill,
+        updatedMessageHash: "0x",
       },
+      messageHash: "0x",
```

Comment:
These should probably be `ZERO_BYTES` for correctness.

Reply:
https://github.com/across-protocol/sdk/pull/940/commits/bd0a140df8bbbc6b16ca697b67e9f51b963d9ec1